jcm/model_test.py

```diff
@@ -283,10 +283,10 @@ def test_speedy_model_default_statistics(self):
 
         model, predictions = run_default_speedy_model(save_interval=30.)
         pred_ds = model.predictions_to_xarray(predictions)
-        pred_ds_monthly = pred_ds.isel(time=-1).mean(dim='lon') # zonal monthly means, take the last month
+        pred_ds_monthly = pred_ds.isel(time=-1).mean(dim={'lon', 'lat'}) # global monthly mean, take the last month
```

Comment:
Technically we should weight these averages but since we're only comparing across runs it doesn't really matter. Maybe add a todo?